contracts/nft/deprecated_nft_test_2.ral

```diff
@@ -0,0 +1,12 @@
+Contract DeprecatedNFTTest2(
+  collectionId: ByteVec,
+  uri: ByteVec
+) {
+  pub fn getTokenUri() -> ByteVec {
+    return uri
+  }
+
+  pub fn getCollectionId() -> ByteVec {
```

Comment:
What would happen if the return type is boolean?

Reply:
Added more tests now, thanks!

Reply:
LGTM.